crates/napi/Cargo.toml

```diff
@@ -13,6 +13,7 @@ edition.workspace = true
 [dependencies]
 ast-grep-core.workspace = true
 ast-grep-config.workspace = true
+ast-grep-language.workspace = true
```

Comment:
ast-grep-langauge has a lot of unnecessary language dependencies. say, tree-sitter parsers. 
I think we can skip the dependency here by implementing(copying) logic to `FrontEndLanguage`